jvcproj/README.md

```diff
@@ -0,0 +1,251 @@
+# JVC D-ILA Control
+
+With this plugin you can control JVC D-ILA projectors over TCP by using the "JVC External Control Command Communication Specification" and transfer gammatables generated with jvcprojectortools.
+
+## Notes
+I use the plugin to control my JVC DLA- X5000 and transfer raw gammatables generated with jvcprojectortools by arvehj. Many thanks to arvehjs groundwork so i could have a look in the communication procedure!
+You can use all operation commands given in the "JVC External Control Command Communication Specification". Wrong or unknown commands will be ignored by the plugin and the projector. Most of the commands won`t be acknowledged when the projector has no input signal. Request commands are not yet supported.
```

Comment:
You should use simple quotes, not back ticks (could lead to problems when rendering this page).